src/test/java/com/pgs/booking/controller/PassengerDtoMapperTest.java

```diff
@@ -0,0 +1,85 @@
+package com.pgs.booking.controller;
+
+import com.pgs.booking.mappers.PassengerDtoMapper;
+import com.pgs.booking.model.Passenger;
+import com.pgs.booking.model.dto.PassengerDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+class PassengerDtoMapperTest {
+
+    private PassengerDtoMapper passengerDtoMapper = new PassengerDtoMapper();
+    private List<PassengerDto> passengerDtoList = new ArrayList<>();
+
+    @BeforeEach
+    void init() {
+        passengerDtoMapper = new PassengerDtoMapper();
+        passengerDtoList = new ArrayList<>();
+    }
+
+    @Test
+    @DisplayName("Mapper passenger list test successful")
+    void mapToPassengersDto() {
```

Comment:
This method doesn't test any mapper functionality. In fact, it doesn't even call it at all.